production.rb

```diff
@@ -89,9 +89,10 @@
   config.action_mailer.delivery_method = :smtp
 
   config.action_mailer.smtp_settings = {
-    address: ENV['SMTP_HOST'],
-    port:    Integer(ENV.fetch('SMTP_PORT', '25'), 10),
-    domain:  ENV['FTP_HOSTNAME'],
+    address:          ENV['SMTP_HOST'],
+    port:             Integer(ENV.fetch('SMTP_PORT', '25'), 10),
+    domain:           ENV['FTP_HOSTNAME'],
+    enable_starttls:  false,
```

Comment:
Could we also use an environment variable with a default value for this?
```suggestion
    enable_starttls: unless ENV['SMTP_DISABLE_STARTTLS'] then true else false end,
```

Reply:
This means we need an additional environment variable, of course, but we keep the default behaviour.